content/blog/using-helm-and-pulumi-to-define-cloud-native-infrastructure-as-code/index.md

```diff
@@ -42,7 +42,7 @@ Specifically, we:
 If you have the Azure command line, try [running the example](https://github.com/pulumi/examples/tree/master/classic-azure-ts-aks-mean)
 with `pulumi up`!
 
-{{< highlight javascript >}}
+```javascript
```

Comment:
This makes the linter happy (since it understands that this is a code block), and has no impact on the rendered page.